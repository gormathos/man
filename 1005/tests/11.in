-9 -5 2 4 -3
-1 -5 -2 -2 -5
8 -9 -3 10 6
