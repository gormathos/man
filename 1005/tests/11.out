3
-9 -5 4
-1 -5 -2
8 -9 10
838
